addons/mail/static/src/components/emoji/emoji.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.Emoji" owl="1">
+        <span class="o_Emoji" t-on-click="ev => emojiListView.onClickEmoji(ev)" t-att-title="props.emoji.description" t-att-data-source="props.emoji.sources[0]" t-att-data-unicode="props.emoji.unicode" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
I don't understand the `onClick`  here, I don't think we need the arrow function anymore

Reply:
Changes done